### PR TITLE
Add instructions on how to login with Plex account that has 2FA

### PR DESCRIPTION
**Pull request by scottrobertson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ Then, run `python3 main.py`.
 At first run, you will be asked to setup Trakt and Plex access.
 Follow the instructions, your credentials and API keys will be stored in
 `.env` and `.pytrakt.json` files.
+
+If you have 2 Factor Authentication enabled on Plex, you can append the code to your password.
```
Comment (glensc) [inline README.md:51]: include example perhaps?

Comment (scottrobertson) [reply README.md:51]: I'm not really sure what examples I can give. You literally just add the 2fa code to the end of your password when logging in.

Comment (glensc) [reply README.md:51]: is there documentation of where you got this from? add such link?

Comment (scottrobertson) [reply README.md:51]: https://github.com/pkkid/python-plexapi/issues/609

https://support.plex.tv/articles/two-factor-authentication/
See "Old Third-Party Apps & Tools"

I guess we could add `<password><verification code>`?

Comment (glensc) [reply README.md:51]: yes, do so, and here's the direct link:
- https://support.plex.tv/articles/two-factor-authentication/#toc-1:~:text=Old%20Third%2DParty%20Apps%20%26%20Tools

**Review by twolaw — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (glensc) [reply README.md:51]: Created followup:
- https://github.com/Taxel/PlexTraktSync/pull/125

**Pull request merged**